### ISSUE TITLE
'Test framework quite unexpectedly' when all test passed 

### ISSUE THREAD

**Issue by piotr-mazur** [lilbaek/webstorm-testcafe#14]
PyCharm Professional Edition 2020.3
I used runner 
tests passed, 
TestCafe terminated without providing any result
![2021-03-23_11-14-06](https://user-images.githubusercontent.com/10757286/112130622-f714fd00-8bc8-11eb-82a6-d3cff90fdaaa.png)

**Comment by barretts** [comment]
@piotr-mazur were you ever able to resolve this? annoying at the least

**Comment by robert-wloch-iits** [comment]
I've just come across that as well. And I can narrow it down.
We've just created a git project which references our repositories as submodules. One such submodule repository contains our e2e tests.

When I open that repository/folder/project directly in Intellij and I right click a fixture and run it then testcafe runs green, finishes with exit code 0 and I can expand the test results.

When I open the master repository and in Intellij and I right click a fixture and run it then testcafe runs green, finishes with exit code 0, but prints "Test framework quit unexpectedly" as shown in the screenshot. Additionally, Intellij displayed on the bottom right for a short time a note saying something like the testcafe runner didn't return a result.

When using the master repository, I had to adapt the run config and point to the working directory. To make the test run. Apart from that no changes have been made between the two Intellij instances of the exact same project on my disk.

IntelliJ: IntelliJ IDEA 2021.3.2 (Ultimate Edition)
TestCafe runner for Webstorm: 1.3.6
macOS: 12.3.1
